structuralcodes/codes/ec2_2023/_section5_materials.py

```diff
@@ -651,21 +651,21 @@ def fctd(_fctk_5: float, _k_tt: float, gamma_C: float) -> float:
         fctk_5 (float): the 5% mean concrete tensile strength fractile in MPa
         _k_tt (float): the factor for considering the effect of high sustained
             loads and of time of loading on concrete tensile strength
-        gamma_C (float): partial factor of concrete
+        gamma_c (float): partial factor of concrete
 
     Returns:
         float: the design tensile strength of concrete in MPa
 
     Raises:
         ValueError: if fctk_5 is less than 0
-        ValueError: gamma_C is less than 1
+        ValueError: gamma_c is less than 1
```

Comment:
why for fctd, ValueError if gamma_c is less than 1, while for fcd ValueError if gamma_c is less than 0?

Reply:
Good point. This was implemented in #24. I have updated it in a53611c.

Reply:
And I updated similarly for `gamma_s` in bde6d60.